.github/workflows/build.yml

```diff
@@ -11,22 +11,26 @@ on:
 
 jobs:
     tests:
-        runs-on: ubuntu-latest
+        runs-on: ubuntu-18.04
 
-        name: "PHP ${{ matrix.php }}, MySQL ${{ matrix.mysql }}, Sylius ${{ matrix.sylius }}"
+        name: "Sylius ${{ matrix.sylius }}, PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
 
         strategy:
             fail-fast: false
             matrix:
                 php: [7.4, 7.3]
+                symfony: [^4.4, ^5.2]
+                sylius: [~1.8.0, ~1.9.0@beta]
```

Comment:
Why we are lock on `1.8.0` ?

Reply:
We are not locked on 1.8.0, `~1.8.0` means `>=1.8.0 <1.9.0`
https://getcomposer.org/doc/articles/versions.md#tilde-version-range-